src/core/CCRandom.mli

```diff
@@ -145,3 +153,11 @@ val (<*>) : ('a -> 'b) t -> 'a t -> 'b t
 
 val run : ?st:state -> 'a t -> 'a
 (** Using a random state (possibly the one in argument) run a generator *)
+
+(** {6 Random generator testing } *)
```

Comment:
Ok for a `(**/**)` section here. Concerning unit tests, did you mean something more in deep than [this short one](https://github.com/c-cube/ocaml-containers/pull/54/files#diff-f4ae21cd7c4ce81d67547aeac4ce2209R220)?